packages/grafana-ui/src/types/datasource.ts

```diff
@@ -281,6 +281,7 @@ export interface ExploreQueryFieldProps<
 
 export interface ExploreStartPageProps {
   onClickExample: (query: DataQuery) => void;
+  mode?: any;
```

Comment:
This could use a comment on what `mode` could be used for.

Reply:
Think mode property can be removed for now based on https://github.com/grafana/grafana/pull/17521#issuecomment-500870094